goagen/gen_swagger/swagger.go

```diff
@@ -347,6 +354,24 @@ func New(api *design.APIDefinition) (*Swagger, error) {
 	return s, nil
 }
 
+func tagsFromDefinition(mdatas []design.MetadataDefinition) (tags []Tag, tagNames []string, err error) {
+	for _, mdata := range mdatas {
+		if mtags, found := mdata["tags"]; found {
+			tagObjs := []Tag{}
```

Comment:
The more "idiomatic" way of doing this is `var tagObjs []Tag`. `append` takes care of creating the slice if it's `nil`.